spec/features/activity_pages/activity_pages_spec.rb

```diff
@@ -12,16 +12,36 @@
   context 'as authenticated user' do
     describe 'index page', js: true do
       let(:trackable) { create(:issue) }
+      let(:trackable_card) { create(:card) }
       let(:user) { create(:user) }
-
+      let(:second_user) { create(:user) }
       let(:create_activities) do
-        50.times do
+        35.times do
           activity = Activity.create(
             user: user,
             trackable_type: trackable.class,
             trackable_id: trackable.id,
             action: 'update',
-            created_at: Time.current - ((1..5).to_a.sample.days)
+            created_at: Time.current - 1.month
+          )
+        end
+        10.times do
+          activity = Activity.create(
+            user: second_user,
+            trackable_type: trackable_card.class,
+            trackable_id: trackable_card.id,
+            action: 'update',
+            created_at: Time.current
+          )
+        end
+
```

Comment:
```suggestion
```
or add a line between L27 and L28, let's be consistent.

Reply:
added a line at L29 to make it consistent. does it look better now?